differ/mutationDiffer.go

```diff
@@ -699,7 +701,18 @@ func (dw *DifferWorker) diff() {
 	case base.MutationCompareTypeBodyAndMeta:
 		includeBody = true
 	}
-
+	srcUUID, err := hlv.UUIDtoDocumentSource(dw.differ.sourceBucketUUID)
+	if err != nil {
+		// This error is very unusual, therefore panic
+		dw.logger.Errorf("Error in converting the bucket UUID %v to required HLV format. err: %v", dw.differ.sourceBucketUUID, err)
+		panic("bucketUUID convertion error")
```

Comment:
Embed "err" in the panic string
`panic(fmt.Sprintf(...`

Reply:
Noted